registry/spec.md

```diff
@@ -12,20 +12,23 @@ automation and tooling.
 - [Notations and Terminology](#notations-and-terminology)
   - [Notational Conventions](#notational-conventions)
   - [Terminology](#terminology)
-- [Attributes and Extensions](#attributes-and-extensions)
-- [Registry APIs](#registry-apis)
-  - [Retrieving the Registry Model](#retrieving-the-registry-model)
-  - [Retrieving the Registry](#retrieving-the-registry)
-  - [Managing Groups](#managing-groups)
-  - [Managing Resources](#managing-resources)
-  - [Managing versions of a Resource](#managing-versions-of-a-resource)
-- [Endpoint Registry](#endpoint-registry)
-  - [Endpoints](#group-endpoints)
-  - [Groups](#group-groups)
-  - [Definition](#resource-definitions)
-- [Schema Registry](#schema-registry)
-  - [Schemagroups](#group-schemagroups)
-  - [Schemas](#resource-schemas)
+- [Registry Formats and APIs](#registry-formats-and-apis)
+  - [Attributes and Extensions](#attributes-and-extensions)
+  - [Registry APIs](#registry-apis)
+    - [Retrieving the Registry Model](#retrieving-the-registry-model)
+    - [Retrieving the Registry](#retrieving-the-registry)
+    - [Managing Groups](#managing-groups)
+    - [Managing Resources](#managing-resources)
+    - [Managing versions of a Resource](#managing-versions-of-a-resource)
+- [CloudEvents Registry](#cloudevents-registry)
+  - [Schema Registry](#schema-registry)
```

Comment:
If this is under a "CE Registry" will some folks think that it only applies to CEs and it's not a general purpose schema registry?

Reply:
That is a very justified question. The "CloudEvents Registry" is probably just a profile of this that is scoped to the "CloudEvents" format. 

We basically have
* A generic registry model that would work for all sorts of schema- and metaschema-like documents
* A concrete schema registry
* A concrete registry for event and messaging envelope definitions
* A concrete registry for eventing and messaging endpoints

There's a "conformance trail" (a set of MUST rules) through those that would make up a "CloudEvents registry", but you can just as well make up a conformance trail that results in an "MQTT registry"

Reply:
I still think Schema Registry should not be under the CE TOC entry.